gulpfile.js

```diff
@@ -4,7 +4,7 @@
 var gulp = require('gulp');
 var log = require('gulp-util').log;
 var sourcemaps = require('gulp-sourcemaps');
-var os = require('os');
+var process = require("process");
```

Comment:
I tried it out on node 0.12.7 and the "process" package doesn't seem to exist there.